templates/page.html

```diff
@@ -7,8 +7,8 @@ <h1>{{ page.title }}</h1>
 {% endblock content %}
 
 {% block prev_link %}
-    {% if page.previous %}
-        <a class="previous" href="{{ page.previous.permalink }}"><</a>
+    {% if page.next %}
+        <a class="previous" href="{{ page.next.permalink }}"><</a>
```

Comment:
having the next page link have a class or `previous` feels wrong. Next/Previous are correct for sections and the first subsection, can we keep the right semantics by changing the ordering without changing the sidebar?